frontend/ios/DroidKaigi 2019/data/AnnouncementRepository.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AnnouncementRepository.swift
+//  DroidKaigi 2019
+//
+//  Created by 佐々木美穂 on 2019/02/03.
+//
+
+import Foundation
+import ioscombined
+import RxSwift
+
+final class AnnouncementRepository {
+    
+    func fetch() -> Single<[AnnouncementResponse]> {
+        return ApiComponentKt.generateDroidKaigiApi()
+        .getAnnouncementsAsync(lang: LangParameter.en)
```

Comment:
I'm not sure how to get default language, so I just set it static...
I'd like to know how to get the default language🙏